### PR TITLE
[Docs] ant documentation

### PR DESCRIPTION
**Pull request by luarss**
Update documentation for consistency in formatting

@dralabeing 

https://openroad--3696.org.readthedocs.build/en/3696/main/src/ant/README.html

### CODE REVIEW

File: src/ant/README.md [hunk 1]
```diff
@@ -22,16 +22,33 @@ Antenna violations can be repaired after global routing with the `repair_design`
...
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-verbose` | report all antenna calculations for violating nets |
+| `-net` | check antennas on said net |
+
+## Regression tests
+
+There are a set of regression tests in `/test`. For more information, refer to this [section](../../README.md#regression-tests).
```
Comment (vvbandeira) [inline src/ant/README.md:45]:
```suggestion
There are a set of regression tests in `./test`. For more information, refer to this [section](../../README.md#regression-tests).
```

File: src/ant/README.md [hunk 2]
```diff
@@ -2,8 +2,8 @@
 
 This tool checks antenna violations and generates a report to indicate
 violated nets. See LEF/DEF 5.8 Language Reference, Appendix C,
-"Calculating and Fixing Process Antenna Violations" for a description of
-antenna violations.
+"Calculating and Fixing Process Antenna Violations" (p.389) 
+for a [description](http://coriolis.lip6.fr/doc/lefdef/lefdefref/lefdefref.pdf) of antenna violations.
```
Comment (vvbandeira) [inline src/ant/README.md:6]:
```suggestion
for a [description](https://coriolis.lip6.fr/doc/lefdef/lefdefref/lefdefref.pdf) of antenna violations.
```

File: src/ant/README.md [hunk 3]
```diff
@@ -22,16 +22,33 @@ Antenna violations can be repaired after global routing with the `repair_design`
...
 ```
 
-- `-verbose` report all antenna calculations for violating nets
-- 'net' check antennas on one net
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-verbose` | report all antenna calculations for violating nets |
+| `-net` | check antennas on said net |
```
Comment (rovinski) [inline src/ant/README.md:41]:
```suggestion
| `-net` | check antennas on the specified net |
```

**Comment by dralabeing** [comment]
> Update documentation for consistency in formatting
> 
> @dralabeing
> 
> https://openroad--3696.org.readthedocs.build/en/3696/main/src/ant/README.html

This looks good. Except for the link into regression tests. Please fix this for all tool PR's for this section.

File: src/ant/README.md [hunk 4]
```diff
@@ -22,16 +22,35 @@ Antenna violations can be repaired after global routing with the `repair_design`
...
-```
 
-- `-verbose` report all antenna calculations for violating nets
-- 'net' check antennas on one net
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-verbose` | Report all antenna calculations for violating nets. |
+| `-net` | Check antennas on specified net. |
```
Comment (vvbandeira) [inline src/ant/README.md:43]:
```suggestion
| `-net` | Check antennas on the specified net. |
```

**Review by vvbandeira — CHANGES_REQUESTED** [review]
Remove `[-report_file report_file]` from `sta::define_cmd_args "check_antennas"` at `AntennaChecker.tcl`

File: src/ant/README.md (hunk 4 above)
Comment (luarss) [reply src/ant/README.md:43]: Can I clarify if you mean to remove all instances that has -report_file in the `check_antennas` function? Or just remove in the `define_cmd_args` section

Comment (vvbandeira) [reply src/ant/README.md]: Remove only from the `sta:: define_cmd_args`.

**Review by vvbandeira — APPROVED** [review]

**Pull request merged**